### PR TITLE
fix: populate cwd and assistant_response in Chronicle session store

### PR DESCRIPTION
**Pull request by digitarald**
Two fields in the Chronicle local session store were always empty. This fixes both with a consumer-only approach (no OTel producer changes) and adds schema caveats for remaining data gaps.

## Changes

**`cwd`** — set from `vscode.workspace.workspaceFolders[0]` during session init.

**`assistant_response`** — `truncateForOTel` produces invalid JSON for long responses (cuts mid-string, appends suffix). The new `extractAssistantResponse()` handles this:
1. Fast path: JSON.parse for non-truncated input
2. Fallback: substring extraction from the fixed JSON prefix + JSON unescape
3. Stored truncated to 1000 chars

**Schema caveats** — updated `_getSchemaDescription()` local branch to warn the LLM that:
- `agent_name` / `agent_description` may be empty for older sessions
- `summary` may contain raw JSON — prefer JOINing with `turns.user_message`
- `assistant_response` may be empty for older sessions
- `session_files` / `session_refs` may be empty for older sessions

## What's NOT fixed (remaining from #312292)

- `agent_name` population — the code reads `GenAiAttr.AGENT_NAME` from spans but this attribute may not be set by all producers
- `agent_description` — no known source for this field
- `summary` containing raw JSON — requires changes to the summarization pipeline
- `session_files` / `session_refs` empty — file/ref tracking was added but may need producer-side work

## Testing

- 9 new test cases in `extractAssistantResponse.spec.ts` (valid JSON, truncated JSON, JSON escape handling, edge cases)
- All chronicle tests pass, TypeScript compiles clean

Partially addresses #312292

### CODE REVIEW

**Review by vijayupadya — DISMISSED** [review]

File: extensions/copilot/src/extension/intents/node/chronicleIntent.ts [hunk 1]
```diff
@@ -407,8 +407,8 @@ Use the session_store_sql tool to run queries. Start with a broad query, then dr
 Use \`now() - INTERVAL '1 day'\` for date math, \`ILIKE\` for text search.
 Always JOIN sessions with turns to get session content — do not rely on sessions.summary alone.`
 			: `Available tables (SQLite syntax — local):
-- **sessions**: id, cwd, repository, branch, summary, host_type, agent_name (who created the session, e.g. 'vscode', 'cli', 'CCA', 'CCR'), agent_description, created_at, updated_at
-- **turns**: session_id, turn_index, user_message, assistant_response, timestamp. The richest source of what actually happened — contains the user's prompts and the assistant's replies.
+- **sessions**: id, cwd (workspace folder path), repository, branch, summary, host_type, agent_name (who created the session, e.g. 'vscode', 'cli', 'CCA', 'CCR'), agent_description, created_at, updated_at
+- **turns**: session_id, turn_index, user_message, assistant_response (truncated to ~1000 chars — a summary of the assistant's reply, not the full response), timestamp. The richest source of what actually happened — contains the user's prompts and the assistant's replies.
```
[the remark this answers is not in the document]
Comment (digitarald) [reply extensions/copilot/src/extension/intents/node/chronicleIntent.ts]: Fixed — reworded to "first ~1000 characters of the assistant reply, with an ellipsis if truncated".

File: extensions/copilot/src/extension/chronicle/common/sessionStoreTracking.ts [hunk 2]
```diff
@@ -141,3 +141,63 @@ export function extractRepoFromMcpTool(toolArgs: unknown): string | undefined {
...
+	// which is 63 chars. The content value follows, terminated by the truncation marker.
+	if (!outputMessagesRaw.includes(OTEL_TRUNCATION_MARKER)) {
+		return undefined;
+	}
+	const contentPrefix = '"content":"';
+	const contentStart = outputMessagesRaw.indexOf(contentPrefix);
+	if (contentStart === -1) {
+		return undefined;
+	}
+	const textStart = contentStart + contentPrefix.length;
```
[the remark this answers is not in the document]
Comment (digitarald) [reply extensions/copilot/src/extension/chronicle/common/sessionStoreTracking.ts]: Fixed — fallback now anchors to `"type":"text","content":"` instead of bare `"content":"`, so tool_call content fields are skipped. Dropped the misleading 63-char comment. Added test verifying tool_call content is skipped in the truncated path.

**Review by vijayupadya — APPROVED** [review]
